en/docs/guides/adaptive-auth/adaptive-auth-with-function-lib.md

```diff
@@ -197,4 +197,10 @@ Follow the instructions below to delete a function library.
     ```tab="Example"
     ageModule.getAge(birthday)
     ```
+    
+!!! info "Related Topics"
+    - [Concept: Adaptive-Authentication](TODO:insert-link-to-concept)
+    - [Guide: Configure Adaptive Authentication for an Application](../../guides/adaptive-auth/configure-adaptive-auth)
+    - [Guide: Ensure Assurance with Authentication Context Class Reference (ACR)  and Authentication Method Reference (AMR)](../../guides/adaptive-auth/work-with-acr-amr)
```

Comment:
can we shorten this to "Ensure Assurance with ACR/AMR" like we have titled it in the mkdocs.yml ?